### PR TITLE
Set default throughput based on account's workload type

### PR DESCRIPTION
**Pull request by asier-isayas**
[Preview this branch](https://cosmos-explorer-preview.azurewebsites.net/pull/2021)

Customers will have the option to set the default throughput based on the account's workload type.

**Learning/DevTest Cohort** - Autoscale container is set to 1000 RU/s instead of 4000 RU/s
**Prod Cohort** - Autoscale container remains at 4000 RU/s

In the event an account does not have a workload type, the default throughput remains same as before.

### CODE REVIEW

**Comment by asier-isayas** [comment]
> @asier-isayas please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by asier-isayas** [comment]
> > @asier-isayas please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> > ```
> > @microsoft-github-policy-service agree [company="{your company}"]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > > Options:
> > > 
> > > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > > 
> > > ```
> > > @microsoft-github-policy-service agree
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > 
> > > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > > 
> > > ```
> > > @microsoft-github-policy-service agree company="Microsoft"
> > > ```
> > 
> > 
> > Contributor License Agreement
> 
> @microsoft-github-policy-service agree [company="Microsoft"]

@microsoft-github-policy-service agree

File: src/Common/DatabaseAccountUtility.ts [hunk 1]
```diff
@@ -15,3 +17,9 @@ function isPrivateEndpointConnectionsEnabled() {
 export function isPublicInternetAccessAllowed(): boolean {
   return !isVirtualNetworkFilterEnabled() && !isIpRulesEnabled() && !isPrivateEndpointConnectionsEnabled();
 }
+
+export function getWorkloadType(): WorkloadType {
```
Comment (jawelton74) [inline src/Common/DatabaseAccountUtility.ts:21]: What does this return when no workload tag is present ? Can you please add unit tests for this ? Maybe we need a 'None' workload type enum ?

**Review by sunghyunkang1111 — DISMISSED** [review]

**Review by vchske — APPROVED** [review]

**Review by jawelton74 — APPROVED** [review]
